plugins/backend-app/package-lock.json

```diff
@@ -0,0 +1,5754 @@
+{
```

Comment:
**issue**: hmm we shouldn't be making lock files for packages: they should all be managed from the root workspace. not sure why these have been generated by npm!?